### PR TITLE
THORN-1890: added maven dependencies to fraction manifest

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: src/main/java/org/wildfly/swarm/plugin/utils/ChecksumUtil.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            while ((len = in.read(buf)) >= 0) {
+                md.update(buf, 0, len);
+            }
+
+            return toHex(md.digest()).toLowerCase();
+        }
+    }
+
+    private static String toHex(byte[] digest) {
+        return DatatypeConverter.printHexBinary(digest);
```
Comment (Ladicek) [inline src/main/java/org/wildfly/swarm/plugin/utils/ChecksumUtil.java]: `javax.xml.bind` will disappear from the JDK soon enough :-)

File: src/main/java/org/wildfly/swarm/plugin/process/FractionManifestGenerator.java [hunk 2]
```diff
@@ -66,6 +70,10 @@ public FractionMetadata apply(FractionMetadata meta) {
                         .map(DependencyMetadata::toString)
                         .collect(Collectors.toList())
             );
+            put("maven-dependencies",
```
Comment (Ladicek) [inline src/main/java/org/wildfly/swarm/plugin/process/FractionManifestGenerator.java:75]: Could you add a comment that the previous `dependencies` and `transitive-dependencies` are "module dependencies per the JBos Modules `module.xml` declarations" or something like that? Just for people who might be coming to the code in the future.

**Comment by kenfinnigan** [comment]
add to whitelist

File: src/main/java/org/wildfly/swarm/plugin/utils/ChecksumUtil.java (hunk 1 above)
Comment (michalszynkiewicz) [reply src/main/java/org/wildfly/swarm/plugin/utils/ChecksumUtil.java]: replaced with my own code

File: src/main/java/org/wildfly/swarm/plugin/process/FractionManifestGenerator.java (hunk 2 above)
Comment (michalszynkiewicz) [reply src/main/java/org/wildfly/swarm/plugin/process/FractionManifestGenerator.java:75]: done

**Review by Ladicek — APPROVED** [review]

**Pull request merged**